app/src/main/java/com/sun_asterisk/moviedb_50/screen/MainActivity.kt

```diff
@@ -0,0 +1,20 @@
+package com.sun_asterisk.moviedb_50.screen
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.sun_asterisk.moviedb_50.R
+import com.sun_asterisk.moviedb_50.screen.home.ContainerFragment
+
+class MainActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+        supportFragmentManager.beginTransaction().replace(
+                R.id.mainFrameLayout,
+                ContainerFragment()
+            )
+            .commit()
```

Comment:
những phần như này, bạn nên tạo 1 hàm thực hiện chức năng navigate này
nếu có thể sẽ viết chúng vào BaseActivity 
```suggestion
        replaceFragment(id = R.id.mainFrameLayout, fragment = ContainerFragment())
```